data/json/items/chemicals_and_resources.json

```diff
@@ -958,7 +955,7 @@
     "name": "handful of hickory nuts",
     "name_plural": "handfuls of hickory nuts",
     "description": "A handful of hard nuts from a hickory tree, still in their shell.",
-    "spoils_in": "180 days",
+    "//spoils_in": "180 days",
```

Comment:
`spoils_in` should be valid node too.

Reply:
`spoils_in` is only valid for comestibles.  This is a generic item.